scripts/setup-credentials.sh

```diff
@@ -73,14 +73,6 @@ auth:
 
 github_pr_creator_org: "10gen"
 
-# Do not edit below this line
 expansions:
-  github_app_key: |
+  github_app_key: "$GITHUB_APP_KEY"
 EOF
-
-# Write the GitHub app key to a file for easier formatting
-echo "$GITHUB_APP_KEY" > app_key.txt
-# Linux and MacOS friendly command to add 4 spaces to the start of each line
-sed -i'' -e 's/^/    /' app_key.txt
-# Append the formatted GitHub app key to the creds.yml file
-cat app_key.txt >> creds.yml
```

Comment:
were creds.yml and app_key.txt not being used?

Reply:
I reverted this back to what it originally was - making a YAML file by hand for a script setup is less than ideal but unless we want to convert it to JSON or something, it's needed due to how YAML handles strings with newlines + caring about leading whitespace syntax.